[2024-04-26T14:02:16.412+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-26T14:02:16.483+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres.create_postgres_table scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-04-26T14:02:16.496+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres.create_postgres_table scheduled__2024-04-01T00:00:00+00:00 [queued]>
[2024-04-26T14:02:16.497+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-26T14:02:16.593+0000] {taskinstance.py:2327} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-04-01 00:00:00+00:00
[2024-04-26T14:02:16.602+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=506) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-26T14:02:16.604+0000] {standard_task_runner.py:63} INFO - Started process 510 to run task
[2024-04-26T14:02:16.601+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres', 'create_postgres_table', 'scheduled__2024-04-01T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmpx7qgc86j']
[2024-04-26T14:02:16.605+0000] {standard_task_runner.py:91} INFO - Job 47: Subtask create_postgres_table
[2024-04-26T14:02:16.689+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres.create_postgres_table scheduled__2024-04-01T00:00:00+00:00 [running]> on host 0ceb125076fc
[2024-04-26T14:02:16.847+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stan' AIRFLOW_CTX_DAG_ID='dag_with_postgres' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T00:00:00+00:00'
[2024-04-26T14:02:16.849+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-26T14:02:16.883+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
        
[2024-04-26T14:02:16.904+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-26T14:02:16.916+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-26T14:02:16.934+0000] {sql.py:457} INFO - Running statement: 
            CREATE TABLE if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
        , parameters: None
[2024-04-26T14:02:16.944+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-04-26T14:02:16.951+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-26T14:02:16.968+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres, task_id=create_postgres_table, execution_date=20240401T000000, start_date=20240426T140216, end_date=20240426T140216
[2024-04-26T14:02:17.022+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-26T14:02:17.062+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-26T14:02:17.065+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
